wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -33,18 +33,20 @@ inline BenchmarkPose interpolate(const PoseMeasurement &m1,
                                  const TimePoint &t) {
     auto w2 = 1.0 * (t - m1.time_point) / (m2.time_point - m1.time_point);
 
-    auto m1inverse = m1.value.rotation;
-    m1inverse.invert();
+    auto m1inverse = inverse(m1.value.rotation);
 
-    auto relative = m2.value.rotation * m1inverse;
-    Vec3 wvec = w2 * relative.logMap();
+    auto relative = eval(log(m2.value.rotation * m1inverse));
```

Comment:
Can be replaced with `m2.value.rotation - m1.value.rotation` I think

log(a * inverse(b)) == a (box minus) b